src/overlays/actors/ovl_Boss_02/z_boss_02.c

```diff
@@ -1664,7 +1664,7 @@ void func_809DD934(Boss02* this, GlobalContext* globalCtx) {
 
         case 1:
             if ((this->unk_1D14 < 80U) && (D_809E0420 != 0) &&
```

Comment:
Could you remove all the `U`s in this file? It matches fine without them.

Reply:
I've marked them all down. There is quite a few more `u`/ `U`'s throughout the codebase, so will do them all in my next general cleanup.